gradle/libs.versions.toml

```diff
@@ -169,7 +169,7 @@ jsoup = "org.jsoup:jsoup:1.20.1"
 appyx_core = { module = "com.bumble.appyx:core", version.ref = "appyx" }
 molecule-runtime = "app.cash.molecule:molecule-runtime:2.1.0"
 timber = "com.jakewharton.timber:timber:5.0.1"
-matrix_sdk = "org.matrix.rustcomponents:sdk-android:25.4.22"
+matrix_sdk = "org.matrix.rustcomponents:sdk-android:0.0.1-element-call-SNAPSHOT"
```

Comment:
This implies, that we need to manually rebuild once we add a commit to the to-device branch right?

Reply:
Yes, there's no way to just auto-upgrade the SDK binary here without building and publishing it first.